qiskit/synthesis/discrete_basis/gate_sequence.py

```diff
@@ -129,6 +157,23 @@ def to_dag(self):
 
         return dag
 
+    def _to_dag(self, adjust_phase: bool):
```

Comment:
See comment above; maybe this is not needed and we can just use the above `to_dag` method (with the `adjust_phase` fix).

Reply:
So now I could delete the new functions `_to_circuit` and `_to_dag` but see the reply to the comment above.